onnxmltools/utils/float16_converter.py

```diff
@@ -13,21 +13,23 @@ def npfloat16_to_int(np_list):
     return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in np_list]
 
 
-def float16_converter(model_path):
+def float16_converter(onnx_file, new_onnx_file):
     """
-    Convert TensorProto type float to float16 in ONNX model.
-    :param model_path: ONNX model path and name
-    :return model: converted ONNX model
+    Convert ONNX model from TensorProto type float to float16.
+    :param onnx_file: ONNX model path and name
+    :param new_onnx_file: new ONNX model path and name
 
     Example:
 
     ::
 
         from onnxmltools.utils import float16_converter
-        model_path = 'c:/winmlperf_coreml_SqueezeNet_prerelease.onnx'
-        model = float16_converter(model_path)
+        onnx_file = 'c:/winmlperf_coreml_SqueezeNet_prerelease.onnx'
```

Comment:
Do we have a better example name? Maybe change `c:/winmlperf_coreml_SqueezeNet_prerelease.onnx` to `example.onnx` and `c:/winmlperf_coreml_SqueezeNet_prerelease_float16.onnx` to `example_float16.onnx`?

Reply:
will do